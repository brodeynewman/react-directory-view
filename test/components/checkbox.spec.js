import React from 'react';
import renderer from 'react-test-renderer';
import CheckBox from '../../src/components/checkbox/Checkbox';

describe('CheckBox', () => {
  it('CheckBox renders correctly', () => {
    const tree = renderer
      .create(<CheckBox />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
