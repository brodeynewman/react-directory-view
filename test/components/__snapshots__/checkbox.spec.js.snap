// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox CheckBox renders correctly 1`] = `
<input
  checked={undefined}
  onChange={undefined}
  style={
    Object {
      "cursor": "pointer",
      "marginRight": 10,
    }
  }
  type="checkbox"
/>
`;
